### DISCUSSION TITLE
Time Series | Adding Custom Metrics

### DISCUSSION THREAD

**Discussion by ngupta23** [pycaret/pycaret#3134] [FAQs]
Refer to the following notebook on how to add custom metrics for time series - [Notebook](https://gist.github.com/ngupta23/410152d2104c73d814cd25b934be11bb)

**Comment by moezali1** [comment]
PyCaret 4.0 sunset the legacy time-series module (sktime/prophet integration), so this FAQ no longer maps to a supported code path.

If you still need time-series forecasting, use sktime / prophet / statsforecast directly — those projects have surpassed what the PyCaret wrapper offered.

For 4.0 docs, see https://pycaret.org/docs and the cleanup notice at https://github.com/pycaret/pycaret/issues/4177.

Closing as outdated.
